sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -1,15 +1,14 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.6 (2021-11-09)
```

Comment:
The date should be of today/tomorrow.
Depending on whenever you will release from release pipeline.

Reply:
Fixed.